flags.css

```diff
@@ -95,19 +95,38 @@
     background-color: rgb(255, 217, 0)
 }
 #sweden>.vertical {
-    width: 12.5%;
+    width: 12.5vmin;
     height: 100%;
     left: 33%;
```

Comment:
rather than doing it this way, I would transform: translateX(-50%) to keep it consistent.